components/common/pkg/kafka/health/health_test.go

```diff
@@ -0,0 +1,198 @@
+package health
+
+import (
+	"fmt"
+	"github.com/kyma-incubator/knative-kafka/components/common/pkg/log"
+	"github.com/stretchr/testify/assert"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"testing"
+)
+
+// Test Constants
+const (
+	testHttpPort    = "8089"
+	testHttpHost    = "localhost"
+	livenessPath    = "/healthz"
+	readinessPath   = "/healthy"
+)
+
+// Set Up Test Logger
+var logger = log.TestLogger()
+
+// Test Struct That Implements The HealthInterface Functions
+type testStatus struct {
+	server *Server
+}
+
+// Mock Function That Returns The Mock Status "Alive" Flag
+func (ts *testStatus) IsAlive() bool {
+	return ts.server.IsAlive()
+}
+
+// Mock Function That Returns True For Readiness Flag
+func (ts *testStatus) IsReady() bool {
+	return true
+}
+
+// Mock Status For Starting Health Server
+var mockStatus testStatus
+
+func getTestHealthServer() *Server {
+	health := NewHealthServer(testHttpPort, &mockStatus)
```

Comment:
Idiomatic Go doesn't have Get or Is in getters.. see https://golang.org/doc/effective_go.html#Getters

Reply:
Altered as suggested